core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -104,11 +93,9 @@ public Configuration(Properties props) {
 
   /**
    * Default constructor.
-   *
-   * Most clients will call this constructor to allow default loading of properties to happen.
    */
   public Configuration() {
```

Comment:
agree. Let's see how far we can go in this PR. Too many references of this constructor (especially in test), and I don't want to further enlarge the footprint of this PR.